### PR TITLE
Leave logs

### PR DESCRIPTION
**Pull request by aremmell**
- Using --leave-logs causes log files to not be deleted after the test suite runs
- Logs are now sent to `./logs/` instead of the working directory
- We need to delete `./logs/*` on clean, and if the `logs` dir isn't present on make tests, we need to mkdir it

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,7 +24,7 @@ docs/xml/
 index.md
 
 # test artifacts
-*.log
```
Comment (johnsonjh) [inline .gitignore:27]: I'd leave *.log in there, just in case, or if ever going back to an older commit.  Can't hurt.

**Review by johnsonjh — APPROVED** [review]
Looks good - I'd do #125 first

**Comment by aremmell** [comment]
> Looks good - I'd do #125 first

Yeah, just waiting on Travis, then merging

**Comment by aremmell** [comment]
I am not sure why it failed before–I think the lack of mkdir for logs... I will stop the build on the first failure this time if it fails again.

**Pull request merged**